### ISSUE TITLE
Enhance IA & Navigation

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network-pulse#863]
We have several overlapping initiatives that include revised IA and nav. These include a more rich homepage with featured tags, a persistent search box, and a box about pulse that appears on first time use.

While we will try to land some small iterations in this direction, we're likely to hit some interdependency. 

This ticket is to track the overlapping work and keep it unblocked. There's still some work that remains on goals, outcomes, and design. 

![image](https://user-images.githubusercontent.com/53411/33093435-f1b1a958-ceca-11e7-96e0-c20aa3b0a19f.png)

**Comment by kristinashu** [comment]
This include a few chunks of work:
- move Search to main nav https://github.com/mozilla/network-pulse/issues/861
- remove Featured, Latest, Issues, and Favs from main nav
  - integrate Featured, Latest, Issues, Tags... into a some sort of sub nav https://github.com/mozilla/network-pulse/issues/859
  - integrate Favs into profile view https://github.com/mozilla/network-pulse/issues/860

Smaller things include:
- update "Signup / Signin" to just Signin 
  - I think this should be dependent on adding the intro banner with Signup CTA
- update "+" CTA to "Add new"

**Comment by gideonthomas** [comment]
superseded by refresh
